cli/start.ts

```diff
@@ -8,6 +8,7 @@ Usage:
 if the <dir> is empty, the current directory will be used.
 
 Options:
+    --host           The address at which the server is to be started.
```

Comment:
You should probably change this to `-h, --host` because it can also use `options.h`

Reply:
so we also need to update `cli.ts` to make `-h` option pass, now it will print help message with `-h`.

Reply:
I would advise against using  `-h` flag for the hostname as it may cause confusion. Instead I suggest that `-hn, --hostname` respectively to avoid any confusion.
```suggestion
    -hn, --hostname  The address at which the server is to be started.
```